### PR TITLE
nb_NO fastlane translations

### PR DESCRIPTION
**Pull request by comradekingu**
I think these help with the listing system on official F-Droid.
There is a requirement to have a bunch of translations to be included in "Latest".

Is a copy of changelogs and images needed?

### CODE REVIEW

File: fastlane/metadata/android/nb-NO/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- Søk
+- Endre navn på programmer og skjul dem
+- Merknad for tilgang til dine favoritter
+- Åpne programmet med vannrett dragning
+- Snarveisstøtte (nettprogrammer)
+- Eksporter og importer innstillinger
+- Hjelp og endringslogg
+
+2/ Tilpasningsvalg
+- Drakt og bakgrunnsstøtte
```
[the remark this answers is not in the document]
Comment (falzonv) [reply fastlane/metadata/android/nb-NO/full_description.txt:20]: Now this has been committed:
https://github.com/falzonv/discreet-launcher/commit/9b606c174bb54b84a623df6d1eec07efd589209c#diff-b62aa6a9e4586317572968b4d1e9813b4751825418d9053a0ae28ee3e75d07f9

[the remark this answers is not in the document]
Comment (comradekingu) [reply fastlane/metadata/android/nb-NO/full_description.txt]: v

File: fastlane/metadata/android/nb-NO/full_description.txt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- Påtvunget stående modus
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply fastlane/metadata/android/nb-NO/full_description.txt]: Fixed :)

**Comment by falzonv** [comment]
Hello,

Thank you for the updates!
I am confused because I just published v5.2.0 without seeing that you updated this PR in parallel (saw it only now).

So although it will be merged now, the new fastlane files will appear in F-Droid only for the next release, sorry...

Best regards.

**Pull request merged**